adding reference to Metrika node monitoring in the monitoring-nodes page (#1305)
onflow/flow: docs/content/node-operation/monitoring-nodes.mdx

sidebar_title: Node Monitoring
---

> <Callout type="info" title="Logs">

Author: Callout do not work on the new doc site

Reviewer: It seems like the text still appears, just without any extra styling. So no need to rush and update other `Callout`s.

| consensus_compliance_sealed_height         | Latest height sealed by this node; should increase at a constant rate. |
| consensus_hotstuff_cur_view                | Current view of the HotStuff consensus algorith; Consensus/Collection only; should increase at a constant rate. |
| consensus_hotstuff_timeout_seconds         | How long it takes to timeout failed rounds; Consensus/Collection only; values consistently larger than 5s are abnormal. |

## Monitoring a Flow node using Metrika Monitoring

Metrika has developed the Flow node monitoring service and is the recommended way of monitoring a Flow node.

> It uses the logs and metrics published by the node and provides access to private node-specific dashboards at no extra-costs.

Reviewer: ```suggestion
It is a free tool that uses the logs and metrics published by the node and provides access to private node-specific dashboards.
```
